### PR TITLE
Fix on indenting and show comments on the steps we are doing 

### PR DESCRIPTION
**Pull request by sumn2u**
Fixed indent and other suggestions made as per [https://github.com/travis-ci/docs-travis-ci-com/pull/624](https://github.com/travis-ci/docs-travis-ci-com/pull/624) .

### CODE REVIEW

**Comment by sumn2u** [comment]
@MariadeAnton can you have a look now ?

**Comment by sumn2u** [comment]
@plaindocs  can you have a look on it ?

**Comment by plaindocs** [comment]
Yep - will do.

File: user/languages/javascript-with-nodejs.md [hunk 1]
```diff
@@ -121,6 +121,54 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework. This is an example using different ember versions.
+
```
Comment (plaindocs) [inline user/languages/javascript-with-nodejs.md]: A couple of nitckpicky edits please:

* a blank line after the heading

Reword to something like
> You can build your Ember applications on Travis CI. The default test framework is [`Qunit`](http://qunitjs.com/).  The following example shows how to build and test against different Ember versions.

Comment (plaindocs) [inline user/languages/javascript-with-nodejs.md]: Where does Qunit come into it? Line 168? What if you just want to build not test?

Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: Ember by default has qunit . While serving and also doing test it automatically build distribution version . The quint here is to give further reference to those who want to learn more about it .

**Comment by plaindocs** [comment]
Thanks! :-)

**Pull request merged**